dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -0,0 +1,287 @@
+
+	# Rename references to isql to fbsql
+	# sed vs patch for portability and addtional location changes
+	check_sed "$(sed -i -e 's:"isql :"fbsql :w /dev/stdout' \
+		src/isql/isql.epp | wc -l)" "1" "src/isql/isql.epp" # 1 line
+	check_sed "$(sed -i -e 's:isql :fbsql :w /dev/stdout' \
+		src/msgs/history2.sql | wc -l)" "4" "src/msgs/history2.sql" # 4 lines
+	check_sed "$(sed -i -e 's:--- ISQL:--- FBSQL:w /dev/stdout' \
+		-e 's:isql :fbsql :w /dev/stdout' \
+		-e 's:ISQL :FBSQL :w /dev/stdout' \
+		src/msgs/messages2.sql | wc -l)" "6" "src/msgs/messages2.sql" # 6 lines
+
+	find "${S}" -name \*.sh -print0 | xargs -0 chmod +x || die
+	rm -rf "${S}"/extern/{btyacc,editline,icu} || die
+
+	eautoreconf
+}
+
+src_configure() {
+	filter-flags -fprefetch-loop-arrays
+	filter-mfpmath sse
+
+	econf \
+		--prefix=/usr/$(get_libdir)/firebird \
+		$(use_enable superserver superserver) \
+		$(use_enable debug) \
+		--with-editline \
+		--with-system-editline \
+		--with-system-icu \
+		--with-fbbin=/usr/bin \
+		--with-fbsbin=/usr/sbin \
+		--with-fbconf=/etc/${PN} \
+		--with-fblib=/usr/$(get_libdir) \
+		--with-fbinclude=/usr/include \
+		--with-fbdoc=/usr/share/doc/${P} \
+		--with-fbudf=/usr/$(get_libdir)/${PN}/UDF \
+		--with-fbsample=/usr/share/doc/${P}/examples \
+		--with-fbsample-db=/usr/share/doc/${P}/examples/db \
+		--with-fbhelp=/usr/$(get_libdir)/${PN}/help \
+		--with-fbintl=/usr/$(get_libdir)/${PN}/intl \
+		--with-fbmisc=/usr/share/${PN} \
+		--with-fbsecure-db=/etc/${PN} \
+		--with-fbmsg=/usr/$(get_libdir)/${PN} \
+		--with-fblog=/var/log/${PN}/ \
+		--with-fbglock=/var/run/${PN} \
+		--with-fbplugins=/usr/$(get_libdir)/${PN}/plugins \
+		--with-gnu-ld \
+		${myconf}
+}
+
+src_compile() {
+	MAKEOPTS="${MAKEOPTS/-j*/-j1} ${MAKEOPTS/-j/CPU=}"
+	emake
+}
+
+src_install() {
+	cd "${S}/gen/${PN}" || die
+
+	if use doc; then
+		dodoc "${S}"/doc/*.pdf
+		find "${WORKDIR}"/manuals -type f -iname "*.pdf" -exec dodoc '{}' + || die
+	fi
+
+	doheader include/*
+
+	rm lib/libfbstatic.a
+
+	insinto /usr/$(get_libdir)
+	dolib.so lib/*.so*
+
+	# links for backwards compatibility
+	dosym libfbclient.so /usr/$(get_libdir)/libgds.so
+	dosym libfbclient.so /usr/$(get_libdir)/libgds.so.0
+	dosym libfbclient.so /usr/$(get_libdir)/libfbclient.so.1
+
+	insinto /usr/$(get_libdir)/${PN}
+	doins *.msg
+
+	use client && return
+
+	einfo "Renaming isql -> fbsql"
+	mv bin/isql bin/fbsql
+
+	local bins="fbsql fbsvcmgr fbtracemgr gbak gdef gfix gpre gsec gstat nbackup qli"
+	for bin in ${bins[@]}; do
+		dobin bin/${bin}
+	done
+
+	dosbin bin/fb_lock_print
+	# SuperServer
+	if use superserver ; then
+		dosbin bin/{fbguard,fbserver}
+	# ClassicServer
+	elif use xinetd ; then
+		dosbin bin/fb_inet_server
+	# SuperClassic
+	else
+		dosbin bin/{fbguard,fb_smp_server}
+
+		#Temp should not be necessary, need to patch/fix
+		dosym "${D}"/usr/$(get_libdir)/libib_util.so /usr/$(get_libdir)/${PN}/lib/libib_util.so
+	fi
```

Comment:
Yes, Firebird does not work that way, you MUST choose and go with one. It is how the engine functions and it changes function thus you cannot have both. The new one is a bit different and may allow for different invocation with different config files. But I am not trying to create instances of Firebird. I do not think many even use it, so hard to justify putting that much more work into it, and not sure about the outcome.